Commands/Public/imgur.js

```diff
@@ -0,0 +1,120 @@
+const imgur = require("imgur");
+const { IsURL } = require("../../Modules/Utils/index");
+const { tokens: { imgurClientID } } = require("../../Configurations/auth");
+
+imgur.setClientId(imgurClientID);
```

Comment:
If we're going to have imgur client ids be editable in the server configs, this needs to be a-mooved

Reply:
I'll worry about that when the custom option is available.